commands/roles.js

```diff
@@ -151,6 +151,7 @@ function registerNewRole(client, guildId, newRoleName) {
   const roleData = client.rolesData.ensure(guildId, []);
   roleData.push(newRoleName);
   client.rolesData.set(guildId, roleData);
+  client.rolesData = client.rolesData.sort();
```

Comment:
per this link: https://www.npmjs.com/package/enmap/v/0.3.0#Enmap+sort

`The sort is not necessarily stable. `

Reply:
@I-AM-STEVEe and @BrandonWLee had test discords but I forget how they ran the bot - worst case we just test in production :)

Reply:
I can spin up mine if you need it.  It takes ~10 mins if you want to set up your own server too